tests/spec/unit/fixtures/test-config-3.xml

```diff
@@ -14,6 +14,7 @@
         <preference name="target-device" value="handset" />
         <preference name="deployment-target" value="11.0" />
         <preference name="SwiftVersion" value="4.1" />
+        <preference name="SupportMac" value="true" />
```

Comment:
I'm assuming an accompanying PR on the template will introduce the preference SupportMac with value false, right?

Reply:
Exactly, it will be added in our template!